### DISCUSSION TITLE
Same project approach

### DISCUSSION THREAD

**Discussion by Jersyfi** [orestbida/cookieconsent#466] [General]
Hi @orestbida, while searching i found your project and saw that we have nearly the same approach in our javascript project (cookieconsent vs. cookify). At this moment i am currently working on an react library for this and i was wondering if you are interesed in working together for a project like that? Maybe checkout my repositories [cookify](https://github.com/Jersyfi/cookify) and [react-cookify](https://github.com/Jersyfi/react-cookify)

If you are not interesed its fine and you can delete my discussion here 👌

**Comment by orestbida** [comment]
Hey @Jersyfi, sorry, It's already hard trying to maintain this plugin. Keep up your good work!

**Reply by Jersyfi** [reply]
When you have open task just ask. Awesome project you got there. Keep it goin 👌
